be/src/common/daemon.cpp

```diff
@@ -204,42 +204,110 @@ void Daemon::tcmalloc_gc_thread() {
 #endif
 }
 
+void refresh_process_memory_metrics() {
+    doris::PerfCounters::refresh_proc_status();
+    doris::MemInfo::refresh_proc_meminfo();
+    doris::GlobalMemoryArbitrator::reset_refresh_interval_memory_growth();
+    ExecEnv::GetInstance()->brpc_iobuf_block_memory_tracker()->set_consumption(
+            butil::IOBuf::block_memory());
+}
+
+void refresh_allocator_memory_metrics() {
```

Comment:
rename to refresh_xmalloc_xxxx
because allocator is very similar to our TAllocator